hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/state/tasks/TraceabilityExportTaskTest.java

```diff
@@ -67,6 +68,7 @@
 class TraceabilityExportTaskTest {
     private static final long ENTITY_NUM = 1234L;
     private static final Instant NOW = Instant.ofEpochSecond(1_234_567, 890);
+    private static final Instant OFFSET_NOW = NOW.plusNanos(3L);
```

Comment:
Oh ho!  Per our conversation on Slack I would _definitely_ want a comment here to explain this totally mysterious and completely unmotivated `3ns`.  

Because not only can't you now find the explanation _here_, you can't find it in the class under test either!